guides/micronaut-oauth2-github/groovy/src/main/groovy/example/micronaut/GithubUser.groovy

```diff
@@ -0,0 +1,21 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@CompileStatic
+@Introspected
+class GithubUser {
+
+    final String login
+    final String name
+    final String email
+
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.